app/serializers/web_hook_post_serializer.rb

```diff
@@ -0,0 +1,200 @@
+class WebHookPostSerializer < ApplicationSerializer
```

Comment:
> Yes, I made a copy of code from those serializers so that they can be updated and controlled separately for the webhook events.

Oops I'm sorry but I didn't know we were still keeping so many attributes. Duplicating all these logic is definitely a no-no. Can I get an idea of how many attributes we actually want to exclude from each serializer?